test/functionalTest/cases/1916_fiware_correlator/fiware_correlator_forward_and_notifications.test

```diff
@@ -214,30 +214,30 @@ Date: REGEX(.*)
 04. Make accumulator subscribe to E2/A2 in CPR1
 ===============================================
 HTTP/1.1 201 Created
-Content-Length: 0
-Location: /v2/subscriptions/REGEX([0-9a-f]{24})
-Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
 
 
 
 05. Create E2/A2 in CPR1, using CORRELATOR2
 ===========================================
 HTTP/1.1 201 Created
-Content-Length: 0
-Location: /v2/entities/E2?type=Thing
-Fiware-Correlator: CORRELATOR2
 Date: REGEX(.*)
+Fiware-Correlator: CORRELATOR2
+Location: /v2/entities/E2?type=Thing
+Content-Length: 0
 
 
 
 06. Update E2/A2 in CB, using correlator CORRELATOR3, provoking a forward and after that a notification to accumulator of E2/A2
 ===============================================================================================================================
 HTTP/1.1 200 OK
-Content-Length: 192
-Content-Type: application/json
-Fiware-Correlator: CORRELATOR3
 Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
```

Comment:
```suggestion
Fiware-Correlator: CORRELATOR3
```

Reply:
Note test is failing, se https://github.com/telefonicaid/fiware-orion/actions/runs/4919583948/jobs/8876244677

```
----------- Failing tests ------------------
  o  1492_limit_zero/limit_zero_types.test
  o  1697_more_than_one_bad_input/more_than_one_bad_input.test
  o  1916_fiware_correlator/fiware_correlator_forward_and_notifications.test
```